### PR TITLE
Clean up unnecessary code

### PR DESCRIPTION
**Pull request by guillermosanta**

### CODE REVIEW

**Comment by guillermosanta** [comment]
In this branch I removed the code that was unecessary. I made the following changes:
* I deleted `20210513022752-install_uuid_generate_v4().js `, a file which imported a function to generate uuids, which is no longer necessary. 
* I removed the migration file that removed the devBeans and goldenBeans attributes, and instead removed it from the user model. I was able to do this because the user model has never been migrated before.

File: api/migrations/20210525230251-create-user.js [hunk 1]
```diff
@@ -11,12 +11,6 @@ module.exports = {
         allowNull: false,
         type: Sequelize.STRING,
       },
-      devBeans: {
```
Comment (chungthuang) [inline api/migrations/20210525230251-create-user.js:14]: You can't change migrations that have been applied. If you want to remove these columns from the database table, you need to create a new migration file.

File: api/migrations/20210823001355-remove-beans.js [hunk 2]
```diff
@@ -1,19 +0,0 @@
-"use strict";
```
Comment (chungthuang) [inline api/migrations/20210823001355-remove-beans.js:1]: Same as https://github.com/dev-launchers-sandbox/project__discord-bot/pull/260/files#r805188286.

File: api/migrations/20210525230251-create-user.js (hunk 1 above)
Comment (chungthuang) [reply api/migrations/20210525230251-create-user.js:14]: Looks like this migration hasn't actually be applied to neither the staging nor production database.

File: api/migrations/20210525230251-create-user.js [hunk 3]
```diff
@@ -11,12 +11,6 @@ module.exports = {
...
-        type: Sequelize.INTEGER,
-      },
-      goldenBeans: {
```
[the remark this answers is not in the document]
Comment (guillermosanta) [reply api/migrations/20210525230251-create-user.js:17]: We will be using the beanLog model to quantify the number of devbeans and goldenbeans. They will allow us to obtain extra information such as who gave the bean and at what time

**Pull request merged**